package.json

```diff
@@ -77,7 +77,8 @@
     "@storybook/react-native": "next",
     "@react-navigation/native": "6.0.11",
     "react-native-safe-area-context": "4.3.1",
-    "expo-blur": "~11.2.0"
+    "expo-blur": "~11.2.0",
+    "recyclerlistview": "^4.1.3"
```

Comment:
Do we need this resolution? Looks like FlashList is using `4.1.2`, did you wanted to upgrade to `4.1.3`?

Reply:
yeah, it is needed.
because the `recyclerlistview ` dependency of `Flashlist` is internal, will move it once [this PR](https://github.com/Shopify/flash-list/pull/564) is merged.